homeassistant/components/system_bridge/config_flow.py

```diff
@@ -113,7 +128,7 @@ class ConfigFlow(
 ):
     """Handle a config flow for System Bridge."""
 
-    VERSION = 1
+    VERSION = 2
```

Comment:
Since this is only moving around variables, can we use the MINOR_VERSION instead? This way if someone downgrades to a lower version it could still work. (Assuming they did not update the software yet)